packages/eslint-plugin-pf-codemods/test/rules/v5/applicationLauncher-warn-input.js

```diff
@@ -0,0 +1,18 @@
+const ruleTester = require("../../ruletester");
+const rule = require("../../../lib/rules/v5/applicationLauncher-warn-input");
+
+ruleTester.run("applicationLauncher-warn-input", rule, {
+  valid: [],
```

Comment:
Would we want to add a valid test here (i.e. something with just the app launcher being used, or is being imported from somewhere else, no PF import) to ensure that the message isn't coming up if they are using an ApplicationLauncher component from some other library?